manageiq-operator/deploy/olm-catalog/manageiq-operator/0.0.1/manageiq.org_manageiqs_crd.yaml

```
@@ -187,6 +197,10 @@ spec:
             orchestratorCpuRequest:
               description: 'Orchestrator deployment CPU request (default: no request)'
               type: string
+            orchestratorImage:
+              description: 'Image string used for the orchestrator deployment (default:
+                <OrchestratorImageName>:<OrchestratorImageTag>)'
```

I think this is slightly wrong.

```suggestion
                <OrchestratorImageNamespace>/<OrchestratorImageName>:<OrchestratorImageTag>)'
```